### PR TITLE
Adding SQLCE 3.5 support

### LINKED ISSUE #122: SQLCE 3.5 Support
Hey Erik,

We had a customer asking about SQLCE 3.5 support in EF7. I just wanted to get your thoughts on whether this is something you might tackle?

### PR DESCRIPTION
**Pull request by ErikEJ**
Fixes #122

### CODE REVIEW

File: src/EntityFramework.SqlCe.Legacy.nuspec [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>EntityFramework.SqlCe.Legacy</id>
```
Comment (bricelam) [inline src/EntityFramework.SqlCe.Legacy.nuspec]: We were wonder what you'd call the package. One thought we had was something like `EntityFramework7.SqlServerCe35` and maybe pair it with `EntityFramework7.SqlServerCe40`.

Comment (bricelam) [inline src/EntityFramework.SqlCe.Legacy.nuspec]: We're still working through our own package names... :wink:

Comment (ErikEJ) [inline src/EntityFramework.SqlCe.Legacy.nuspec]: I tried to keep the same naming scheme as was used for the EF6 packages, while I await your final package naming decision! Ideally, I would like EntityFramework.SqlServerCompact and EntityFramework.SqlServerCompact.Legacy, but they were taken by someone called "Microsoft"

Comment (ErikEJ) [inline src/EntityFramework.SqlCe.Legacy.nuspec]: But my package name are by no means final!

**Pull request merged**